Makefile

```diff
@@ -345,24 +345,24 @@ contribs/dist: .build/build-dll.timestamp
 	touch $@
 
 .build/node_modules.timestamp: package.json
-	npm install
-	mkdir -p $(dir $@)
+	npm install --no-optional
+	rm -rf ./node_modules/@types/openlayers
```

Comment:
Why this ?

Reply:
Installed from peer dependency from ol-layerswitcher and that breaks our types

Added as comment :-)

Reply:
:+1: thanks